towhee/cnn_trainer/training_args.py

```diff
@@ -0,0 +1,173 @@
+# Copyright 2020 The HuggingFace Team and 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import socket
+from dataclasses import asdict, dataclass, field
+from datetime import datetime
+from enum import Enum
+from typing import List, Optional
+
+import torch
+
+from towhee.cnn_trainer.utils import logging
+
+logger = logging.get_logger(__name__)
+log_levels = logging.get_log_levels_dict().copy()
+trainer_log_levels = dict(**log_levels, passive=-1)
+
+
+def default_logdir() -> str:
+    """
+    Same default as PyTorch
+    """
+
+    current_time = datetime.now().strftime("%b%d_%H-%M-%S")
+    return os.path.join("runs", current_time + "_" + socket.gethostname())
+
+
+@dataclass
+class TrainingArguments:
+    """
+    Training arguments of CNN trainer.
+    """
```

Comment:
since training args are part of user interface, we'd better add detailed docstrings for documentation